automatic-off-session-payment/process.edn

```diff
@@ -0,0 +1,169 @@
+{:format :v3,
+ :transitions
+ [{:name :transition/enquire,
+   :actor :actor.role/customer,
+   :privileged? true,
+   :actions [
+     {:name :action/create-pending-booking}
```

Comment:
Start the actions array on the same line with the first action:

```
:actions
[{:name :action/action-name...
```

Reply:
Fixed!